scripts/install_dependencies.sh

```diff
@@ -27,9 +27,10 @@ DEPENDENCIES="build-essential \
     qtbase5-dev \
     uuid-dev \
     wget \
-    zip \
     zlib1g \
-    curl"
+    zip \
+    wget \
+    libperl-dev"
```

Comment:
What is the purpose of the perl library?

Reply:
addressed